src/darker/tests/test_main.py

```diff
@@ -14,12 +13,13 @@
 import darker.__main__
 import darker.import_sorting
 from darker.git import RevisionRange
+from darker.tests.conftest import GitRepoFixture
 from darker.utils import TextDocument
 from darker.verification import NotEquivalentError
 
 
 def test_isort_option_without_isort(tmpdir, without_isort, caplog):
-    check_call(["git", "init"], cwd=tmpdir)
+    GitRepoFixture.create_repository(tmpdir)
```

Comment:
Why not just use the `git_repo` fixture directly? It does exactly this (plus chdirs into the repo root).
```python
def test_isort_option_without_isort(git_repo, without_isort, caplog):
    # check_call(["git", "init"], cwd=tmpdir)  # remove this...
    # GitRepoFixture.create_repository(tmpdir)  # ...as well as this
```
The `GitRepoFixture` import should then be removed as well.

Reply:
sure, ended up having to add a docstring to the test and `# pylint: disable=unused-argument` since neither `git_repo` nor `without_isort` end up getting used.